src/db/mysql/models.rs

```diff
@@ -433,7 +432,7 @@ impl MysqlDb {
         Ok(results::GetBsos {
             bsos,
             more,
-            offset: next_offset,
+            offset: next_offset as i64,
```

Comment:
nit I forgot to revert this cast, it's no longer needed

Reply:
Fixed.